### DISCUSSION TITLE
Some Help With Configuration

### DISCUSSION THREAD

**Discussion by bigverm23** [trinib/Adguard-Wireguard-Unbound-Cloudflare#73] [Help]
Ok, so I'm trying to understand this...but, I want to use Unbound for caching and Anonymized DNSCrypt for DoH.

I have DNSCrypt setup and working (according to logs).

For Unbound, do I use DNSCrypt proxy as the forward-zone, or something else like Cloudflare?

and I'm confused about where in this equation, Cloudflare and Stubby come into play?

IF I want to use Stubby as a TLS forwarder: I would use forward-zone: stubby in unbound then, and NOT use DNSCrypt proxy?

**Comment by trinib** [comment]
If using DNSCrypt for DoH. There is no need for cloudflare and stubby programs. Its just there for an alternative option that is more simple than DNSCryp. 

Undound + DNSCrypt
Undound + Cloudflare Tunnel + Stubby

I will make it more clear in readme
